samza-api/src/main/java/org/apache/samza/table/descriptors/RemoteTableDescriptor.java

```diff
@@ -164,8 +165,7 @@ public RemoteTableDescriptor(String tableId) {
    * @return this table descriptor instance
    */
   public RemoteTableDescriptor<K, V> withRateLimiter(RateLimiter rateLimiter,
-      TableRateLimiter.CreditFunction<K, V> readCreditFn,
-      TableRateLimiter.CreditFunction<K, V> writeCreditFn) {
+      TableRateLimiter.CreditFunction<K, V> readCreditFn, TableRateLimiter.CreditFunction<K, V> writeCreditFn) {
```

Comment:
Can you make sure you don't change the original formatting, if a line is not changed. There is a few cases in the PR

Reply:
I see... Seems like my auto format setting is different from the original file... I will change them back.